### DISCUSSION TITLE
Runtime Multiwindow with Next JS

### DISCUSSION THREAD

**Discussion by NRohner** [tauri-apps/tauri#9965] [Q&A]
I'm writing an app using Tauri and Next JS. I would like to open a second window when a user clicks a button in my app but am struggling with the multiwindow. I've read the [multiwindow](https://tauri.app/v1/guides/features/multiwindow) page on the Tauri documentation site but am still a bit lost. In the documentation, it says 

> If you want to create additional pages beyond index.html, you will need to make sure they are present in the dist directory at build time. How you do this depends on your frontend setup.

So my question is, has anyone done this with Next JS? I don't even see a dist directory in my folder structure. Additionally, Next JS uses .tsx files rather than HTML files. Any help or advice here would be greatly appreciated!!

**Comment by FabianLars** [comment]
in my opinion the easiest way is just to create a new route for it and then use something like `/Route` or `/#/Route` (i forgot how nextjs' router works) as the window url.

> I don't even see a dist directory in my folder structure

It's not talking about a literal `dist/` dir, but the path configured in distDir(v1) / frontendDist(v2) in tauri.conf.json
